ipatests/prci_definitions/nightly_master.yaml

```diff
@@ -38,6 +38,222 @@ jobs:
           version: 0.1.9
         timeout: 1800
         topology: *build
+        
+  fedora-28/simple_replication:
+    requires: [fedora-28/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-28/build_url}'
+        test_suite: test_integration/test_simple_replication.py
+        template: *ci-master-f28
+        timeout: 3600
+        topology: *master_1repl
+
+  fedora-28/caless:
+    requires: [fedora-28/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-28/build_url}'
+        test_suite: test_integration/test_caless.py::TestServerReplicaCALessToCAFull
```

Comment:
This test is already present in gating as `fedora-28/test_caless_TestServerReplicaCALessToCAFull`.